src/api/users/users.controllers.ts

```diff
@@ -0,0 +1,86 @@
+import { Request, Response, NextFunction } from "express";
+import { 
+  createUser,
+  deleteUser,
+  getAllUsers, 
+  getUserById,
+  updateUser,
+
+ } from "./users.services";
+
+ // gets all the users from the bd
+ export const getAllUsersController = async (
+  req: Request,
+  res: Response,
+ ) => {
+  try {
+    const users = await getAllUsers()    ;
+    res.status(200).send({ message: 'Users retrieved successfully', data: users });
+  } catch (error: any) {
+    res.status(500).json({message: error.message});
+  }
+ }
+
+ // get single user with id
+
+ export const getUserByIdController = async (
+  req: Request,
+  res: Response,
+ ) =>{
+  try {
+    const { id } = req.params;
+    const user = await getUserById(id);
+    if (!user) {
+      return res.status(404).json({ message: 'User with that ID does not exist' });
+    }
+    
+    res.status(200).json({ message: 'User found!', data: user });
+  } catch (error: any) {
+    res.status(500).json({ message: error.message })
+  }
+ }
+
+ // create new user
+
```

Comment:
delete this break line, the comment should be next to the function, no spaces between

Reply:
resolved